get_issues.py

```diff
@@ -1,54 +1,57 @@
 import requests
 
-def print_issues(project_name):
-    """A function that prints a list of open issues (not pull-request)
+
+def get_issue_list(project_name):
+    """A function that returns a list of open issues (not pull-request)

```

Comment:
```suggestion
    """ Returns the list of open issues (not pull-request)
```

Не нужно писать, что это function, оно и так понятно.